docs/events.md

```diff
@@ -28,14 +28,22 @@ No events are emitted for `Conditions` today (https://github.com/tektoncd/pipeli
   successfully, including post-steps injected by Tekton.
 - `Failed`: this is triggered if the `TaskRun` is completed, but not successfully.
   Causes of failure may be: one the steps failed, the `TaskRun` was cancelled or
-  the `TaskRun` timed out.
+  the `TaskRun` timed out. `Failed` events are also triggered in case the `TaskRun`
+  cannot be executed at all because of validation issues.
 
 ## PipelineRuns
 
 `PipelineRun` events are generated for the following `Reasons`:
 
+- `Started`: this is triggered the first time the `PipelineRun` is picked by the
+  reconciler from its work queue, so it only happens if web-hook validation was
+  successful. Note that this event does not imply that a step started executing,
```

Comment:
Fair point. I called it started because that's the terminology we use in the controller (see `IsStarted`).

Thinking more about it, I would still say that "started" is valid, and that a run is:
- created by the user when it's submitted to etcd
- started when the controller picks it up
- running when it actually starts executing the logic described by the run

Reply:
`Created` sounds reasonable as well, but I agree with @afrittoli we are using terms like `Started`, `Running`, and `Stopping`, `Succeeded`, etc. We could add one more event for `Created` but need to balance the number of events generated, few folks have reported too many events being created (#2842 and #2844).

Reply:
> created by the user when it's submitted to etcd

Yeah, I realized this after posting the comment. I don't feel too strongly about this at all. So, I'm happy to keep `Started` as the Reason.